pandas/tests/indexing/test_partial.py

```diff
@@ -451,7 +451,7 @@ def test_partial_set_empty_frame(self):
         # these work as they don't really change
         # anything but the index
         # GH5632
-        expected = DataFrame(columns=['foo'], index=Index([], dtype='int64'))
+        expected = DataFrame(columns=['foo'], index=Index([]))
```

Comment:
I think we want to change this test. IIUC, previously the expected index was an empty Int64Index, which would compare equal to an empty RangeIndex. Now it'll be an empty object-dtype index.

cc @jreback